python/cugraph/cugraph/tests/test_balanced_cut.py

```diff
@@ -97,26 +97,24 @@ def test_edge_cut_clustering_with_edgevals(graph_file, partitions):
     assert cu_score < rand_score
 
 
-# Test to ensure DiGraph objs are not accepted
+# Test to ensure Directed objs are not accepted
 # Test all combinations of default/managed and pooled/non-pooled allocation
-
-
-def test_digraph_rejected():
-    gc.collect()
-
-    df = cudf.DataFrame()
-    df["src"] = cudf.Series(range(10))
-    df["dst"] = cudf.Series(range(10))
-    df["val"] = cudf.Series(range(10))
-
-    with pytest.deprecated_call():
-        G = cugraph.DiGraph()
-    G.from_cudf_edgelist(
-        df, source="src", destination="dst", edge_attr="val", renumber=False
-    )
-
-    with pytest.raises(Exception):
-        cugraph_call(G, 2)
+#def test_digraph_rejected():
```

Comment:
Should this be commented out?  Is the intended behavior of balanced cut to raise an exception for directed graphs?

Reply:
I dropped that code.  It was really only testing that an exception was raised when a DiGraph was created